Makefile-az.mk

```diff
@@ -44,6 +47,20 @@ az-mkaks-cilium: az-mkacr ## Create test AKS cluster (with --network-dataplane c
 	az aks get-credentials --name $(AZURE_CLUSTER_NAME) --resource-group $(AZURE_RESOURCE_GROUP) --overwrite-existing
 	skaffold config set default-repo $(AZURE_ACR_NAME).azurecr.io/karpenter
 
+az-mkvnet:
+	az group create --name $(AZURE_RESOURCE_GROUP)-vnet --location $(AZURE_LOCATION) 
+	az network vnet create --name $(AZURE_CLUSTER_NAME)-vnet --resource-group $(AZURE_RESOURCE_GROUP)-vnet --location $(AZURE_LOCATION) --address-prefixes "10.1.0.0/16"
+
+az-mksubnet: 
+	az network vnet subnet create --name $(AZURE_CLUSTER_NAME)-subnet --resource-group $(AZURE_RESOURCE_GROUP)-vnet --vnet-name $(AZURE_CLUSTER_NAME)-vnet --address-prefixes "10.1.0.0/24"
```

Comment:
i prefer to have resources with the same prefix "karp" so that i can later go and do cleanup of the resources but don't feel as strongly for the subnet